tests/auto/tests.xml

```diff
@@ -31,6 +31,9 @@
            <case manual="false" name="webpagefactory">
                <step>cd /opt/tests/sailfish-browser/auto/ &amp;&amp; ./tst_webpagefactory</step>
            </case>
+           <case manual="false" timeout="300" name="logins">
```

Comment:
I'd rather remove other timeouts as well.

Reply:
This is now fixed so the tests cases run in under 90 seconds; I've removed the extended timeout.